AnnaHollandSmith/Interactive_CV
# Add .DS_Store to .gitignore and remove existing .DS_Store files

### Summary

This pull request addresses an issue with untracked `.DS_Store` files cluttering the repository. It adds `.DS_Store` to the `.gitignore` file and removes existing `.DS_Store` files from the repository.

### Changes Made

- Added `.DS_Store` to the `.gitignore` file to prevent future commits of this file.
- Removed all existing `.DS_Store` files from the repository history.

### Context

The presence of `.DS_Store` files can lead to unnecessary clutter and inconsistencies in the repository. By ignoring these files and removing existing ones, we ensure a cleaner and more organized version history.

### How to Test

1. Clone this repository to a local directory.
2. Checkout this branch: `git checkout add-ds-store-ignore`.
3. Verify that `.DS_Store` is added to the `.gitignore` file.
4. Verify that no `.DS_Store` files exist in the repository.
5. Perform necessary testing to ensure that the repository functions as expected.

### Checklist

- [x] Added `.DS_Store` to `.gitignore`
- [x] Removed existing `.DS_Store` files

Thank you for reviewing this pull request. Your feedback is highly appreciated!
